python/src/test/java/org/apache/zeppelin/python/IPythonInterpreterTest.java

```diff
@@ -475,7 +475,7 @@ public void testIPythonFailToLaunch() throws InterpreterException {
       fail("Should not be able to start IPythonInterpreter");
     } catch (InterpreterException e) {
       String exceptionMsg = ExceptionUtils.getStackTrace(e);
-      assertTrue(exceptionMsg, exceptionMsg.contains("No such file or directory"));
+      assertTrue(exceptionMsg, exceptionMsg.contains("java.io.IOException"));
```

Comment:
Why this change ?

Reply:
My system language is German. When I run tests locally, some tests fail because the JVM translates the error message into German. With commit https://github.com/apache/zeppelin/pull/3925/commits/ea099abef02a0b971c863c1bb1b70aa149bb74fc I have removed the part that is translated by the JVM.

Reply:
I still think it is better to check the exact error message instead of a general IOException. I believe there should be a setting to turn off the error message translation

Reply:
I was looking for a solution, but unfortunately I did not find a suitable one. Since this change is only an accessory, I will remove it.